node/cmd/guardiand/admintemplate.go

```diff
@@ -204,6 +207,13 @@ func init() {
 	AdminClientAccountantModifyBalanceCmd.Flags().AddFlagSet(moduleFlagSet)
 	TemplateCmd.AddCommand(AdminClientAccountantModifyBalanceCmd)
 
+	// flags for the core-bridge-set-message-fee command
+	coreBridgeSetMessageFeeFlagSet := pflag.NewFlagSet("core-bridge-set-message-fee", pflag.ExitOnError)
+	coreBridgeSetMessageFeeChainId = coreBridgeSetMessageFeeFlagSet.String("chain-id", "", "Chain ID")
```

Comment:
Does it makes sense to use a uint32 flag instead of a string here?

Reply:
The problem here is that we're using a string as an argument, e.g:

```
build/bin/guardiand template core-bridge-set-message-fee --chain-id base_sepolia --message-fee 0
```